docs/source/conf.py

```diff
@@ -0,0 +1,58 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+
+import stimela
+
+project = 'stimela'
+copyright = '2022, Sphesihle Makhathini, Oleg Smirnov and RATT'
+
+author = stimela.__author__
+release = stimela.__version__
+version = stimela.__version__
+
+# -- General configuration ---------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#general-configuration
+
+extensions = []
+
+templates_path = ['_templates']
+exclude_patterns = []
+
+language = 'ls -lrt'
+
+# -- Options for HTML output -------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output
+
+html_theme = 'alabaster'
```

Comment:
[furo](https://github.com/pradyunsg/furo) is a nice theme but you'll have to add it to the `docs` group mentioned elsewhere in this review.

```suggestion
html_theme = 'furo'
```

I don't think this is critical though.